### ISSUE TITLE
Capture UX flow for Carebot

### ISSUE THREAD

**Issue by livlab** [thecarebot/carebot#21]
from data sourcing to reporting to notifications over time. See #9

**Comment by livlab** [comment]
@hampelm I made good progress figuring out a good way to illustrate the flow of various things. Several questions about timing and what triggers what came up, which was revealing and useful. I wasted some time trying to represent things but went to the whiteboard and stuff started to come out. (yay!) 

It may not look like much now, but this will become something useful to show when and what happens from publishing of a story (tracking begins) to the last time a journalist is notified about a story by Carebot.

![flow1x](https://cloud.githubusercontent.com/assets/1936365/12796687/02b3bc48-ca8e-11e5-97d5-2b9dcbd2822e.jpg)

**Comment by livlab** [comment]
Some observations:
- If there is an (estimated) 4-hour lag for data from Google Analytics, the timing of the first notification with data is tricky, specially since it starts a chain reaction for all subsequent notifications
- Considering that the impact of stories that don't get promoted (either via homepage time or on facebook), is substantially diminished, I wondered whether or not to initiate notifications before promotion happens. I'm leaning towards starting AFTER promotion happens (which would require HP and FB integrations, boo! out of scope!). This would make for a more meaningful set of data (because the story got a chance at success and because it's had at a minimum a couple extra hours to get over the GA data lag/delay.
- Given our focus on a single metric, frequency/volume is a different question than if we have more metrics and other outlets (email, dashboard report). For example, if a journalist is going to get a daily morning email, I wouldn't send a notification first thing because it would redundant. So, if email was in, I'd reduce to 2 notifications on that day, one mid-AM other mid-PM. (illustrated above).
- The sketch so far presumes about 12 notification messages from start to finish. I originally tested with about 9 during my manual alpha test. 4 of those 12 are not data-focused so I don't think that creates a burden/makes Carebot too chatty. We'll find out if that's true.
- As I started to sketch this out, it because clear that if I stuck only to Phase One scope, this would be simplistic, not simple, which is why I included the above considerations. I also included the path of what it would look like with the dashboard/report available at different points and made me think about how it might evolve over time, but I won't focus on that until we get to that point. It's just there so I can speak to it when we go through things.
- Things that are triggers for notifications: tracker started to send data over; story is published on the npr.org homepage (future);  story is posted on NPR facebook (future); time of day (for email).
